modules/_labs/core/react/lib/theming/styled.ts

```diff
@@ -0,0 +1,36 @@
+import {default as emotionStyled, CreateStyled, Interpolation, CSSObject} from '@emotion/styled';
+import {CanvasTheme, ContentDirection} from './types';
+import {useTheme} from './theme';
+import rtlCSSJS from 'rtl-css-js';
+
+const noop = (styles: any) => styles;
+
+// Pulled from https://github.com/emotion-js/emotion/blob/master/packages/styled-base/src/utils.js#L6 (not exported)
+type Interpolations = Array<any>;
+
+function styled<Props>(node: any) {
```

Comment:
Note: I'm moving this over to #272. You'll have to rebase once that's there and make any changes necessary.